recipes-devtools/gie/tensorrt-samples_8.0.1-1.bb

```diff
@@ -0,0 +1,49 @@
+DESCRIPTION = "NVIDIA TensorRT Samples for deep learning"
+LICENSE = "Proprietary"
+
+inherit l4t_deb_pkgfeed cuda
+
+HOMEPAGE = "http://developer.nvidia.com/tensorrt"
+
+L4T_DEB_GROUP = "tensorrt"
+
+SRC_COMMON_DEBS = "\
+    libnvinfer-samples_${PV}+cuda10.2_all.deb;downloadfilename=libnvinfer-samples_${PV}+cuda10.2_all.deb;name=samples;subdir=tensorrt \
+"
+
+SRC_URI:append = " file://0001-Makefile-fix-cross-compilation-issues.patch"
+
+SRC_URI[samples.sha256sum] = "6eadd053f8e840f17ee8ea5cde98b2b170b91a763cc5cf0bda5a8c59a4d5390d"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+LIC_FILES_CHKSUM = "file://../../../share/doc/libnvinfer-samples/copyright;md5=713c2de2adb0f371a903b9fe20431bab"
+
+S = "${WORKDIR}/tensorrt/usr/src/tensorrt/samples"
+
+DEPENDS = "cuda-cudart cudnn tegra-libraries tensorrt-core tensorrt-plugins libglvnd"
+
+EXTRA_OEMAKE = ' \
+    CUDA_INSTALL_DIR="${STAGING_DIR_HOST}/usr/local/cuda-${CUDA_VERSION}" \
+    CUDNN_INSTALL_DIR="${STAGING_DIR_HOST}/usr/lib" \ 
+    TRT_LIB_DIR="${STAGING_DIR_HOST}/usr/lib" \
+    TARGET="${TARGET_ARCH}" \
+'
+
+do_configure() {
+    :
+}
+
+do_install() {
+    install -d ${D}${prefix}/src/tensorrt/bin
+    cp --preserve=mode,timestamps ${S}/../bin/sample_* ${D}${prefix}/src/tensorrt/bin
+    install -d ${D}${prefix}/src/tensorrt/data
+    cp -R --preserve=mode,timestamps ${S}/../data/* ${D}${prefix}/src/tensorrt/data
+}
+
+FILES:${PN} += "${prefix}/src/tensorrt/bin ${prefix}/src/tensorrt/data"
+RDEPENDS:${PN} += "tensorrt-core tensorrt-plugins tegra-libraries libglvnd"
+
+INSANE_SKIP:${PN} += "ldflags"
+
+PACKAGE_ARCH = "${TEGRA_PKGARCH}"
```

Comment:
This PACKAGE_ARCH setting should be removed.  The `cuda` bbclass sets the appropriate value.

Reply:
Done